### ISSUE TITLE
Error creating application gateway. Invalid Json for --servers

### ISSUE THREAD

**Issue by andymoe** [Azure/azure-cli#1619]
When I run this command I get the following error. Note: this happens if I use a single ip for --servers or multiple IPs. I've not tried with a FQDN as that's not a current use case for me.

az network application-gateway create --name an-application-gateway-name --resource-group a-resource-group --location westus --vnet-name a-vnet-name --subnet a-subnet --servers 172.17.1.4 172.17.1.5 --capacity 2 --sku-name Standard_Large --cert-file certfile.pfx --cert-password ACertPassword --http-settings-protocol Https --frontend-port 443 --public-ip-address dev-gateway-api-public-ip

```At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/arm-debug for usage details. {
  "error": {
    "code": "InvalidRequestFormat",
    "message": "Cannot parse the request.",
    "details": [
      {
        "code": "InvalidJson",
        "message": "Error converting value \"{'ip_address': '172.17.1.4', 'fqdn': None}\" to type 'Microsoft.WindowsAzure.Networking.Nrp.Frontend.Contract.Csm.Public.ApplicationGatewayBackendAddress'. Path 'properties.backendAddressPools[0].properties.BackendAddresses[0]', line 1, position 831."
      },
      {
        "code": "InvalidJson",
        "message": "Error converting value \"{'ip_address': '172.17.1.5', 'fqdn': None}\" to type 'Microsoft.WindowsAzure.Networking.Nrp.Frontend.Contract.Csm.Public.ApplicationGatewayBackendAddress'. Path 'properties.backendAddressPools[0].properties.BackendAddresses[1]', line 1, position 876."
      }
    ]
  }
}  Correlation ID: xxxxxxxxxxxxxx
```

**Comment by tjprescott** [comment]
Can you run `pip list` and tell me what version of `azure-mgmt-network` you have?

**Comment by tjprescott** [comment]
The fix for this will be in PR #1606.

**Comment by tjprescott** [comment]
PR #1606 merged.
